manager/assets/modext/widgets/resource/modx.panel.resource.js

```diff
@@ -105,7 +121,7 @@ Ext.extend(MODx.panel.Resource,MODx.FormPanel,{
                 this.rteLoaded = false;
             }
         }
-
+        // console.log("MODx.panel.Resource::setup, isDirty?", this.isDirty());
```

Comment:
This line should be deleted :)

Reply:
Thought I'd tracked down all those logging lines! Fixed.